bubble-wrap-style.yaml

```diff
@@ -4802,6 +4664,7 @@ layers:
                 draw:
                     dots:
                         color: [[4,[0.525,0.765,0.686]],[9,[0.416,0.737,0.631]]]
+        # TODO: This seems broken
```

Comment:
Tile content, or something I can help with?

Reply:
That's a comment from a while ago, not sure what's going on there without some debugging that's beyond this PR.